src/column/number_column.jl

```diff
@@ -54,22 +54,23 @@ Type defining a numeric column and its display settings.
 | `header_name::String` | `field_name` | Override `field_name` as default with custom value. |
 | `visible::Bool` | `true` | Column visibility. |
 | `filter::Bool` | `false` | Enables a filter for column values. |
+| `default_sort::AG_SORT_MODES` | `nothing` (`AG_ASC`, `AG_DESC`) | Initial sorting in the column. |
 | `formatter::AGFormatter` | `nothing` | Formatting of the numbers. |
 | `str_format::String` | `"%s"` | String template for formatting values. The `%s` is replaced with the value. |
 | `width::Integer` | `nothing` | Column width. |
 | `cell_background::String` | `"#fff"` | Cell background. |
 | `rect_background::String` | `"#fff"` | Background color of the rectangle. |
 | `color::String` | `"#000"` | Text color. |
 | `text_align::AG_TEXTALIGN_TYPES` | `AG_CENTER` (`AG_LEFT`, `AG_RIGHT`) | Text alignment. |
-| `equals::Vector{Number}` | `Float64[]` | The values that are highlighted in color in the cell. |
-| `equals_color::String` | `"#ca3c32"` | The color that will be used to highlight values that are equal to `equals`. |
+| `color_map::Dict{Number,String}` | `Dict{Number,String}()` | A dictionary of colors by their values. |
```

Comment:
```suggestion
| `color_map::Dict{Number,String}` | `Dict{Number,String}()` | Dictionary with colors of values (`value` => `color`) |
```

Maybe something like this ?